src/synopsys-action/tools-parameter.ts

```diff
@@ -177,6 +177,8 @@ export class SynopsysToolsParameter {
 
     if (isNullOrEmptyValue(coverityStreamName)) {
       const defaultStreamName = (isPrEvent ? process.env[constants.GITHUB_ENVIRONMENT_VARIABLES.GITHUB_BASE_REF] : process.env[constants.GITHUB_ENVIRONMENT_VARIABLES.GITHUB_REF_NAME]) || ''
+      debug(`Github Base Ref: ${process.env[constants.GITHUB_ENVIRONMENT_VARIABLES.GITHUB_BASE_REF]}`)
+      debug(`Github Ref Name: ${process.env[constants.GITHUB_ENVIRONMENT_VARIABLES.GITHUB_REF_NAME]}`)
```

Comment:
Need to check do we need to log these here.
Since we are logging Coverity project name and Stream name, this will cover these env variables.

Reply:
removed